tests/integration/pages/desktop/scanResults.page.js

```diff
@@ -6,7 +6,7 @@
 class ScanResultsPage {
 
   waitForPageToLoad() {
-    $(".scan-results").waitForExist(5000);
+    $(".scan-results").waitForExist(100000);
```

Comment:
This almost feels like a big leap. I'll assume ms here, since the previous value of "5000" ms is 5 seconds, which meaning this timeout is now 100000ms or 100 seconds or 1.6 minutes, although I concede the screenshots are pretty big, so maybe it takes a bit for the UI to fully bake.

Reply:
I was frequently getting errors with the 5s limit. :/

Reply:
I did this for testing...thanks for catching it.